Sources/Utils/Dropdown/DropdownController.swift

```diff
@@ -20,7 +20,9 @@ class DropdownController: UIViewController {
     }
   }
 
-  var topConstraint: NSLayoutConstraint?
+  var expandedTopConstraint: NSLayoutConstraint?
```

Comment:
What's the idea behind this 🤔

Reply:
all this commits are fixes of rotation bugs) this one eliminates problem when album view appears while rotating from landscape to portait orientation. It is better choice to use two constraints Instead of manually setup view dimension to constraint constant.